keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,337 @@
+# Azure Key Vault Secret client library for Java
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings. Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the secrets and its versions.
+
+Use the secret client library to create and manage secrets.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][azkeyvault_docs] | [Samples][secrets_samples]
+
+## Getting started
+### Adding the package to your project
+
+Maven dependency for Azure Secret Client library. Add it to your project's pom file.
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-keyvault-secrets</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
At some point you are preparing to change this to 1.0.0-preview.1?

Reply:
yeah, better to do it in Core updates PR too.